python/cuda_cccl/setup.py

```diff
@@ -0,0 +1,57 @@
+# Copyright (c) 2024, NVIDIA CORPORATION & AFFILIATES. ALL RIGHTS RESERVED.
+#
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+import os
+import shutil
+
+from setuptools import setup, find_namespace_packages
+
+
+project_path = os.path.abspath(os.path.dirname(__file__))
+cccl_path = os.path.abspath(os.path.join(project_path, "..", ".."))
+cccl_headers = [["cub", "cub"], ["libcudacxx", "include"], ["thrust", "thrust"]]
+__version__ = None
+with open(os.path.join(project_path, "cuda", "cccl", "_version.py")) as f:
+    exec(f.read())
+assert __version__ is not None
+ver = __version__
+del __version__
+
+
+with open("README.md") as f:
+    long_description = f.read()
```

Comment:
this can be moved to pyproject.toml too, ex:
https://github.com/NVIDIA/cuda-python/blob/33b7366e308201f3bca8206ae331e399ac1b3379/cuda_core/pyproject.toml#L65
(`readme` is the new preferred name over `long_description`)

Reply:
Done: commit 40c83898e29cd76c1d240e998d82c71a43dbc0dc